client/rpcserver/handlers.go

```diff
@@ -1584,4 +1654,88 @@ needed to complete a swap.`,
 	stopMarketMakingRoute: {
 		cmdSummary: `Stop market making.`,
 	},
+	stakeStatusRoute: {
+		cmdSummary: `Get stake status. `,
+		returns: `Returns:
+  obj: The staking status.
+    {
+      ticketprice (uint64): The current ticket price in atoms.
+      vsp (string): The url of the currently set vsp (voting service provider).
+      isrpc (bool): Whether the wallet is an RPC wallet. False indicates
+an spv wallet and allows options that can view and set the vsp.
```

Comment:
`allows options that can view ...` `->` `enables options to view ...`

Reply:
fixed